homework-week3.js

```diff
@@ -0,0 +1,196 @@
+}
+
+//2. Arrays!
+
+let favoriteAnimals = ["blowfish", "capricorn", "giraffe"];
+//Add a statement that adds Mauro's favorite animal 'turtle' to the existing array.
+favoriteAnimals.push("turtle");
+
+console.log(favoriteAnimals);    //(4) ["blowfish", "capricorn", "giraffe", "turtle"]
+
+//add 'meerkat', but make sure it will be placed after 'blowfish' and before 'capricorn'.
+favoriteAnimals.splice(1, 0, "meerkat");  // 1: the index of the new element,    0: the number of the elements I want to remove  
+
+console.log("This statement should add the new element 'meerkat' in the second position which has the index 1 without removing any other elements from the existing array.");
+console.log(favoriteAnimals);   //(5) ["blowfish", "meerkat", "capricorn", "giraffe", "turtle"]
+
+//Log the length of the array
+console.log(favoriteAnimals.length);
+console.log("The array has a length of: " + favoriteAnimals.length);   //The array has a length of: 5
+
+// delete 'giraffe' from the array.
+const removed = favoriteAnimals.splice(3, 1);   // I'll delete more elements => I'll use let to declare my variable
+/* 3: the index of the element we want to remove,    
+1: how many times we want to repeat the deletion of elements which will have the index 3 after the first deletion 
+*/
+console.log(favoriteAnimals);    //(4) ["blowfish", "meerkat", "capricorn", "turtle"]
+
+// you want to delete 'meerkat', but you don't know the position or the index of the item in the array, how can you find it?
+favoriteAnimals.indexOf("meerkat");     // we find the index of the element ---- it is : 1
+console.log("The item you are looking for is at index: " + favoriteAnimals.indexOf("meerkat"));
+//removed = favoriteAnimals.splice(1, 1);  // we delete it using .splice(the index, how many times)
+//console.log(favoriteAnimals);
+
+
+// ------ More JavaScript --------
+
+// 3 arguments function that returns the sum
+function sum (a,b,c) {
+    return a+b+c;
+}
+sum (1,2,3);   // 6
+
+//---------colorCar function -----------
+function colorCar (color) {
+    return "a "+ color + " car";   //the input should be a string .... I tried a lot but couldn't use red as a color not as a string????
+} 
+colorCar("red");   //"a red car"
+
+//--------vehicle function (obj)---------
+const personalInfo = {
+    name: "Ghufran", 
+    age: 34,
+};
+function myFunction (obj){      
+    return  console.log(personalInfo.name, personalInfo.age);
+}
+myFunction(personalInfo) // Ghufran 34
+
+
+// ------ vehicleType function-----------
+function vehicleType (color,code) {
+    if (code === 1){
+        return "a " + color + " car";
+    } else {                          
+        return "a " + color + " motorbike";
+    } 
+}
+vehicleType("blue", 2)    //"a blue motorbike"
+
+/*
+if (3 === 3) {
+  console.log("yes");
+} else {
+  console.log("no");
+}
+*/
+const message = (3 === 3) ? "yes" : "no";
+console.log(message);  //      yes
```

Comment:
Very good, this could be a one-liner as well, `console.log(3 === 3 ? 'yes' : 'no')`